pkgs/development/compilers/llvm/10/compiler-rt/default.nix

```diff
@@ -25,6 +25,8 @@ stdenv.mkDerivation {
     "-DCOMPILER_RT_DEFAULT_TARGET_ONLY=ON"
     "-DCMAKE_C_COMPILER_TARGET=${stdenv.hostPlatform.config}"
     "-DCMAKE_ASM_COMPILER_TARGET=${stdenv.hostPlatform.config}"
+  ] ++ lib.optionals (haveLibc && !isMusl) [
```

Comment:
Why is musl different from glibc here?

Reply:
https://gist.github.com/mweinelt/b1357859bf4a2f659a7b793db3ec7051